### ISSUE TITLE
Update CLI help text for invalid option error

### ISSUE THREAD

**Issue by t1m0thyj** [zowe/zowe-cli#2138]
When running a command with an invalid option such as `zowe files ls ds --protocol invalid`, the error message says:
> The value must match one of the following regular expressions:

Although regexes are supported here, in many cases the values are simple strings rather than regexes.
So we want to simplify the wording to say "the following options" instead of "the following regular expressions".

To make this change, wording needs to be updated here and any failing snapshots will need to updated: https://github.com/zowe/zowe-cli/blob/11c8edc6d557799cfcc3d955e0c94bdad059dadd/packages/imperative/src/cmd/src/syntax/SyntaxValidator.ts#L847

**Comment by jace-roell** [comment]
👍
